src/main/java/org/embl/mobie/viewer/projectcreator/ui/ProjectsCreatorPanel.java

```diff
@@ -479,27 +491,24 @@ public void addBdvFormatImageDialog() {
             gd.addMessage( "Note: You can only 'link' to images outside the project folder \n" +
                     " for local projects. 'copy' or 'move' if you wish to upload to s3");
 
-            // TODO - ImageDataFormat.OmeZarr removed from here for now. Add it back when transforms are supported
-            // so the voxel size can be maintained.
-            String[] imageFormats = new String[]{ ImageDataFormat.BdvN5.toString(),
-                    ImageDataFormat.BdvOmeZarr.toString() };
-            gd.addChoice( "Image format", imageFormats, imageFormats[0] );
-
+            gd.addChoice( "Image format", imageFormats, imageFormats[lastImageDataFormatIndex] );
             String[] addMethods = new String[]{ ProjectCreator.AddMethod.link.toString(),
                     ProjectCreator.AddMethod.copy.toString(), ProjectCreator.AddMethod.move.toString() };
-            gd.addChoice("Add method:", addMethods, addMethods[0]);
-            String[] imageTypes = new String[]{ ProjectCreator.ImageType.image.toString(),
-                    ProjectCreator.ImageType.segmentation.toString() };
-            gd.addChoice("Image Type", imageTypes, imageTypes[0]);
-            gd.addCheckbox("Make view exclusive", false );
+            gd.addChoice("Add method:", addMethods, addMethods[lastAddMethodIndex]);
+            gd.addChoice("Image Type", imageTypes, imageTypes[lastImageTypeIndex]);
+            gd.addCheckbox("Make view exclusive", lastExclusive );
 
             gd.showDialog();
 
             if (!gd.wasCanceled()) {
-                ImageDataFormat imageDataFormat = ImageDataFormat.fromString( gd.getNextChoice() );
-                ProjectCreator.AddMethod addMethod = ProjectCreator.AddMethod.valueOf( gd.getNextChoice() );
-                ProjectCreator.ImageType imageType = ProjectCreator.ImageType.valueOf( gd.getNextChoice() );
-                boolean exclusive = gd.getNextBoolean();
+                lastImageDataFormatIndex = gd.getNextChoiceIndex();
+                ImageDataFormat imageDataFormat = ImageDataFormat.fromString( imageFormats[lastImageDataFormatIndex] );
+                lastAddMethodIndex = gd.getNextChoiceIndex();
+                ProjectCreator.AddMethod addMethod = ProjectCreator.AddMethod.valueOf( addMethods[lastAddMethodIndex] );
+                lastImageTypeIndex = gd.getNextChoiceIndex();
+                ProjectCreator.ImageType imageType = ProjectCreator.ImageType.valueOf( imageTypes[lastImageTypeIndex] );
+                lastExclusive = gd.getNextBoolean();
+                boolean exclusive = lastExclusive;
```

Comment:
I think you can simplify this (and the others).

If there is just one 
`private static exclusive = true`
and
`gd.addCheckbox("Make view exclusive", exclusive );`
`exclusive = gd.getNextBoolean();`

then it should work, isn't it?

Reply:
Good point! I simplified it in the last commit